src/Model/Behavior/SequenceBehavior.php

```diff
@@ -259,6 +246,89 @@ public function beforeDelete(Event $event, Entity $entity)
     }
 
     /**
+     * Decrease the position of the entity on the list
+     *
+     * If a "higher" entity exists, this will also swap positions with it
+     *
+     * @param \Cake\Datasource\EntityInterface $entity The entity that is going to be saved.
+     * @return bool
+     */
+    public function moveUp(EntityInterface $entity)
```

Comment:
Can add a second param `$number` similar to that of [TreeBehavior::moveUp()](https://api.cakephp.org/3.4/class-Cake.ORM.Behavior.TreeBehavior.html#_moveUp). Same for the `moveDown()` method.